### PR TITLE
Chores, chores, chores

### PR DESCRIPTION
**Pull request by kargakis**
* Update the toucan contracts submodule to latest commit
* Rename bridge contract from ToucanBridge to ToucanRegenBridge
* Use tco2 interface instead of pulling the implementation
* Replace custom Ownable contract with OpenZeppelin version
* Use same compiler version as the one used by Toucan contracts
* Update deploy script to work
* README updates to document build and deploy processes

### CODE REVIEW

**Comment by kargakis** [comment]
cc @pheuberger

**Review by pheuberger — APPROVED** [review]
looks good to me 👍 clean and straight forward

File: contracts/ToucanRegenBridge.sol [hunk 1]
```diff
@@ -68,14 +65,11 @@ contract ToucanBridge is Ownable, Pausable {
...
+
         totalTransferred += amount;
 
         emit Bridge(msg.sender, recipient, address(tco2), amount);
-
-        require(
-            nctRegistry.checkERC20(address(tco2)),
-            "contract not part of the Toucan NCT registry"
-        );
         tco2.retireFrom(msg.sender, amount);
```
Comment (marceljay) [inline contracts/ToucanRegenBridge.sol:73]: Is this currently still in the spec, or this is just a WIP for testing?

Comment (kargakis) [reply contracts/ToucanRegenBridge.sol:73]: Likely used just for testing, we need to replace with `burn` once that is properly exposed by the TCO2 contracts.

**Pull request merged**